server/statistics/store.go

```diff
@@ -260,24 +261,25 @@ func (s *StoresStats) GetStoresKeysReadStat() map[uint64]float64 {
 // RollingStoreStats are multiple sets of recent historical records with specified windows size.
 type RollingStoreStats struct {
 	sync.RWMutex
-	bytesWriteRate          MovingAvg
-	bytesReadRate           MovingAvg
-	keysWriteRate           MovingAvg
-	keysReadRate            MovingAvg
+	bytesWriteRate          *AvgOverTime
```

Comment:
Why the region stats also need to change to `aot`?  `aot` can't handle noise.

Reply:
The noise caused by leader transferring is already handled by TiKV. Other noise I think will mainly come from short interval heartbeats, which will be solved by `AvgOverTime`.

Reply:
I will revert changes on regions, and do more test. If this works, I will submit another PR.

Reply:
got.